ui/jobs/jobs.js

```diff
@@ -761,6 +761,25 @@ class BuffTracker {
         this.gainAbilityMap[buff.gainAbility] = buff;
       }
     }
+
+    const v5_08 = {
+      trick: {
+        durationSeconds: 10,
+      },
+      peculiar: {
+        durationSeconds: 10,
+      },
+    };
+
+    let buffOverrides = {
+      ko: v5_08,
+      cn: v5_08,
+    };
+
+    for (let key in buffOverrides[gLang.lang]) {
```

Comment:
Instead of gLang.lang, can you use `this.options.Language`?

Reply:
done with it.